ftl/core/deck-config.ftl

```diff
@@ -414,22 +413,22 @@ deck-config-reschedule-cards-warning =
 deck-config-ignore-before-tooltip = 
     If set, reviews before the provided date will be ignored when optimizing & evaluating FSRS parameters.
     This can be useful if you imported someone else's scheduling data, or have changed the way you use the answer buttons.
-deck-config-compute-optimal-weights-tooltip =
-    Once you've done 1000+ reviews in Anki, you can use the Optimize button to analyze your review history,
-    and automatically generate parameters that are optimal for your memory and the content you're studying.
-    If you have decks that vary wildly in difficulty, it is recommended to assign them separate presets, as
-    the parameters for easy decks and hard decks will be different. There is no need to optimize your parameters
-    frequently - once every few months is sufficient.
+deck-config-compute-optimal-weights-tooltip2 =
+    When you click the Optimize button, FSRS will analyze your review history to generate parameters that are 
+    optimal for your memory patterns and the content you're studying. If your decks vary wildly in difficulty, it 
+    is recommended to assign them separate presets, as the parameters for easy decks and hard decks will be different. 
+    You don't need to optimize your parameters frequently - once every few months is sufficient.
     
     By default, parameters will be calculated from the review history of all decks using the current preset. You can
     optionally adjust the search before calculating the parameters, if you'd like to alter which cards are used for
     optimizing the parameters.
-deck-config-compute-optimal-retention-tooltip2 =
-    This tool assumes that you’re starting with 0 learned cards, and will attempt to find the desired retention
-    value that will lead to the most material learnt, in the least amount of time. This number can be used as a
-    reference when deciding what to set your desired retention to. You may wish to choose a higher desired retention,
-    if you’re willing to trade more study time for a greater recall rate. Setting your desired retention lower than
-    the minimum is not recommended, as it will lead to more work without benefit.
+deck-config-compute-optimal-retention-tooltip3 =
+    This tool assumes that you’re starting with 0 learned cards, and will attempt to find the desired retention value 
+    that will lead to the most material learnt, in the least amount of time. To accurately simulate your learning process, 
+    this feature requires a minimum of 400+ reviews. The calculated number can serve as a reference when deciding what to 
```

Comment:
How likely is it that this number will change again? We should not be specific if there is a chance it will change again.

Reply:
This number was not changed by me. Here is the discussion: https://github.com/ankitects/anki/pull/3019#discussion_r1498167555

I don't have a plan to modify it in the future.

Reply:
Are you happy with it at 400? Sorry, we should have checked with you at the time.

Reply:
400 is OK to me. And I plan to make the simulation less sensitive to outliers.

https://github.com/open-spaced-repetition/fsrs-optimizer/issues/107